### PR TITLE
Update Reexport to its full release.

### PR DESCRIPTION
**Pull request by Datseris**
Please consider installing CompatHelper.jl.

### CODE REVIEW

**Comment by gummif** [comment]
I'm not familiar with it, will take a look. But could you specify the requirements like so `Reexport = "0.2, 1"` so that it works with older julia versions?

File: Project.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 [compat]
 DSP = "0.5.1, 0.6"
 FFTW = "0.3, 1"
-Reexport = "0.2"
+Reexport = "1.0"
```
Comment (Datseris) [inline Project.toml:17]:
```suggestion
Reexport = "0.2, 1.0"
```

**Comment by gummif** [comment]
Do you know why this is failing? https://travis-ci.org/github/JuliaDSP/Wavelets.jl/jobs/753172473

**Comment by Datseris** [comment]
no idea

**Comment by gummif** [comment]
I will merge, hopefully only a random CI bug.

**Pull request merged**